Sat, 04 Jan 2020 10:25:06 - INFO - Connect to DB
Sat, 04 Jan 2020 10:25:06 - INFO - TRIZMA DB import
Sat, 04 Jan 2020 10:25:06 - INFO - 123TV DB import
Sat, 04 Jan 2020 11:11:02 - INFO - Connect to DB
Sat, 04 Jan 2020 11:11:02 - INFO - TRIZMA DB import
Sat, 04 Jan 2020 11:11:02 - INFO - 123TV DB import
Sat, 04 Jan 2020 11:30:35 - INFO - Connect to DB
Sat, 04 Jan 2020 11:30:35 - INFO - TRIZMA DB import
Sat, 04 Jan 2020 11:30:35 - INFO - 123TV DB import
Sat, 04 Jan 2020 11:50:58 - INFO - Connect to DB
Sat, 04 Jan 2020 11:50:58 - INFO - TRIZMA DB import
Sat, 04 Jan 2020 11:50:58 - INFO - Failed on TRIZMA DB import at: 2020-01-04_10-15-56, for following calling party: 4924069878311, for agent id: trz_divankovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_divankovic',                                  '4924069878311','','Nicht übermittelt','4924069878311_Saturday, January 4, 2020_12:50:53 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Sat, 04 Jan 2020 11:50:58 - INFO - 123TV DB import
